content/CPU_Instruction_Set.md

```diff
@@ -1,5 +1,5 @@
-Tables below specify the mnemonic, opcode bytes, clock cycles, affected
-flags (ordered as znhc), and explanation. The timings assume a CPU
+Tables below specify the mnemonic, encoding, cycles, affected
```

Comment:
Those are actually clocks (4 MiHz), not cycles (1 MiHz). The distinction, as explained below, is very important.

Reply:
Then should the table headings be modified also to say "Clock cycles" instead of "cycles"?

Reply:
Please do!